flutter/pubspec.yaml

```
@@ -17,10 +17,10 @@ dependencies:
   package_info_plus: ^1.0.0
 
 dev_dependencies:
-  build_runner: ^2.1.5
+  #build_runner: ^2.1.5
```

Don't we need it at all? if so, let's remove it.

---

That's still needed in order to create the mocks. So this line should be reverted. I'm currently not able to use the latest Flutter version, so I had to temporarily remove it. 
```suggestion
  build_runner: ^2.1.5
```